boilerplate/fullstack/next-app-dir-multitenancy/app/auth/[[...path]]/page.tsx

```diff
@@ -11,11 +11,6 @@ import { ChangeTenantsButton } from "../../../app/components/changeTenantsButton
 import { TenantSelector } from "../../../app/components/TenantSelector";
 
 export default function Auth() {
-    if (typeof window === "undefined") {
```

Comment:
how come we don't require the ` const [loaded, setLoaded] = useState(false);` stuff here? It's there in the non multi tenancy example.

Reply:
there we check that the session context has been loaded - which achieves the same thing.